test/benchmark/test-benchmark-buffer.js

```diff
@@ -26,6 +26,7 @@ runBenchmark('buffers',
                'source=array',
                'type=',
                'value=0',
-               'withTotalLength=0'
+               'withTotalLength=0',
+               'difflen=false'
```

Comment:
Super minor optional nit: The rest of the options are alphabetized. Can this go after line 14 intstead?